### ISSUE TITLE
how to isolate root cause black-holed request

### ISSUE THREAD

**Issue by github-germ** [MISP/MISP#4544]
2.4.102

general question to help diagnose root cause of black-holed client requests...

we see requests that are deemed black-holed in `error.log`. however, we could use some help diagnosing the root cause as the stack trace is not clear to us.

is there some documentation we can read on this, or some further debugging we can do to ind the cause and fix

thanks!!

**Comment by github-germ** [comment]
this issue is a roadblock to us rolling out a new release to our production MISP host. any help on isolating root cause on black-holed requests would be greatly appreciated. thx...

**Comment by iglocska** [comment]
What functionality is generating the black holes?

**Comment by github-germ** [comment]
@iglocska -- thanks for getting in touch!! `/feeds/edit` ... ah, i see a colleague has posted more details on https://github.com/MISP/MISP/issues/4551

**Comment by github-germ** [comment]
the same code (python or curl) works on my MISP instance, but does not on another test instance (staging for production deployment). both are running 2.4.102, but clearly something in the config must differ. trying to isolate and fix. thx.

**Comment by github-germ** [comment]
btw, other REST requests work (i.e. no black-holed). seems to be isolated to `feeds`, e.g.
`attributes/restSearch` works fine.

**Comment by github-germ** [comment]
two additional thoughts:
1. `feeds/previewIndex/1` (with the _corrected_ `body` as below) works. so why is `feeds/edit` black-holed?
2. could this be related in any way to https://github.com/MISP/MISP/issues/4400 ? That went unchanged, but I do know with 2.4.102 we must enclose `feed` `body` requests within a `Feeds` key, e.g.

* WORKS:
```
{"Feeds": []}
```
* DOES NOT WORK (which is what the default template loads):
```
[]
```
which yields:
```
{"name":"Invalid JSON input. Make sure that the JSON input is a correctly formatted JSON string. This request has been blocked to avoid an unfiltered request.","message":"Invalid JSON input. Make sure that the JSON input is a correctly formatted JSON string. This request has been blocked to avoid an unfiltered request.","url":"\/feeds\/previewIndex\/1"}
```

**Comment by github-germ** [comment]
@iglocska ~~~ digging, dig, dug...

ok, we are on 2.4.102. this was a bug i reported and you fixed after 2.4.102 as per https://github.com/MISP/MISP/issues/4142#issuecomment-462520325

and i _forgot_ that i had patched my MISP instance with that on top of 2.4.102 (**mea culpa**), and the staging test server did not have that patch.

case closed.

**### SORRY!!!!**
